config/puma.rb

```diff
@@ -21,7 +21,7 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
+workers ENV.fetch("WEB_CONCURRENCY") { 1 }
```

Comment:
I lowered this because Puma crashes locally when you quit the server if you run two processes. But on Heroku we should probably set this variable to at least 2 so that we can process requests faster.